pkg/api/server_test.go

```diff
@@ -25,17 +27,85 @@ import (
 	"github.com/apache/apisix-ingress-controller/pkg/config"
 )
 
+const (
+	certFileName = "cert.pem"
```

Comment:
You may try to use `ioutil.TempFile` to generate different files, instead of specifying filename by yourself.